frontend/src/components/navigation/AppNavBar.tsx

```diff
@@ -15,16 +15,18 @@ function AppNavBar() {
 
   const ManagePropertyDropDown = () => (
     <NavDropdown title="Manage Property" id="manage-property-dropdown">
-      <NavDropdown.Item>Acquisition Project</NavDropdown.Item>
-      <NavDropdown.Item>Disposition Project</NavDropdown.Item>
-      <NavDropdown.Item>Request Exemption</NavDropdown.Item>
+      <NavDropdown.Item onClick={() => history.push('/submitProperty')}>
```

Comment:
What does this do?

Reply:
this is how the user gets to the submitProperty page. Uses router history so that the back button works.